README.md

```diff
@@ -59,13 +60,49 @@ optional arguments:
   --perform            Perform the task
 ```
 
-## Minimal Slack permission scopes required
+## Permission Scopes needed
+
+The permissions to grant depend on what you are going to use the script for.
+Grant the permissions below depending on your use.
+
+Beyond granting permissions, if you wish to use this script to delete
+messages or files posted by others, you will need to be an [Owner or
+Admin](https://get.slack.help/hc/en-us/articles/218124397-Change-a-member-s-role)
+of the workspace.
+
+#### Deleting messages from public channels
 
 - `channels:history`
 - `channels:read`
-- `chat:write:bot`
+- `chat:write:user
```

Comment:
please add the missing backtick at the end. also missing in the other `chat:write:user` scopes

Reply:
Done.